src/lgcompressed.jl

```diff
@@ -0,0 +1,41 @@
+struct LGCompressedFormat <: AbstractGraphFormat end
+
+function savegraph(fn::AbstractString, g::AbstractGraph, gname::AbstractString,
+    format::LGCompressedFormat)
+    io = open(fn, "w")
+    try
+        io = GzipCompressorStream(io)
+        return savegraph(io, g, gname, LightGraphs.LGFormat())
+    catch
+        rethrow()
+    finally
+        close(io)
```

Comment:
So here you close the `GzipCompressorStream`. Will this also close the file that you opened above?

Reply:
I'm assuming so, since this is how [the example in the CodecZlib README](https://github.com/bicycle1885/CodecZlib.jl#usage) shows it.

Reply:
You are right, I just tested that.